source/common/mongo/proxy.h

```diff
@@ -8,6 +8,8 @@
 
 #include "envoy/access_log/access_log.h"
 #include "envoy/common/time.h"
+#include "envoy/event/dispatcher.h"
```

Comment:
is this still needed?

Reply:
nope

Reply:
still needed, but I moved that to the `cc` file.